samples/meetings-details-tab/nodejs/Readme.md

```diff
@@ -0,0 +1,71 @@
+# Details Tab
+
+This sample app illustrates the implementation of Details Tab in Meeting. User can create a poll and post poll in meeting chat and participants can submit their feedback in Meeting.
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org) version 10.14 or higher
+
+    ```bash
+    # determine node version
+    node --version
+    ```
+
+      
+ - [Ngrok](https://ngrok.com/download) (Only for devbox testing) Latest (any other tunneling      software       can also be used)
+    ```bash
+
+     # run ngrok locally
+    ngrok http -host-header=localhost 4000
+    ```
+- [Teams](https://teams.microsoft.com) Microsoft Teams is installed and you have an account
+
+## To try this sample
+
+1. Clone the repository
+
+    ```bash
+    git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+    ```
+
+    - In a terminal, navigate to `samples/meetings-details-tab/nodejs`
+
+        ```bash
+        cd samples/meetings-details-tab/nodejs
+        ```
+
+    - Install modules and Start the bot
+
+        ```bash
+        npm run server
+        ```
+
+    - Start client application
+
+        ```bash
```

Comment:
Is it possible to keep this in a single application host for simplicity of sample code?